### PR TITLE
Add runbook for LowVirtControllersCount alert

### PR DESCRIPTION
**Pull request by sradco**
Signed-off-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

File: docs/runbooks/LowVirtControllersCount.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+```
+kubectl -n kubevirt logs <pod-name>
+```
+```
+kubectl -n kubevirt describe pod/<pod-name>
+```
+
+## Mitigation
+
+There can be several reasons, identify the root cause and fix it.
```
Comment (iholder-redhat) [inline docs/runbooks/LowVirtControllersCount.md:28]: This is not very helpful :)

I would look at:
- Maybe not enough memory on the cluster?
- Nodes are down?
- API server is overloaded? (e.g. Scheduler has a lot of work and therefore is not 100% available)
- Networking issues?

Maybe others can provide better places to look at

**Review by iholder-redhat — COMMENTED** [review]
Hey @sradco, thanks a lot!!

One general feedback is that I would avoid writing specific technical details, such as "we usually have 1 replica". `LowVirtControllersCount` should generally alert about a logical problem and `Low` can be different according to dynamic factors.

In other words: if we change the number of replicas to 3 - or even determine it on runtime - this doc would be outdated.

File: docs/runbooks/LowVirtControllersCount.md (hunk 1 above)
Comment (sradco) [reply docs/runbooks/LowVirtControllersCount.md:28]: @rmohr Can you please help with this?

Comment (sradco) [reply docs/runbooks/LowVirtControllersCount.md:28]: @xpivarc @rmohr  Can you please suggest some mitigation options for this alert?

**Comment by sradco** [comment]
> To make this runbook compatible with both upstream and downstream, would you consider exporting the namespace and using it after (for example, kubevirt for upstream, openshift-cnv for downstream)? I see in other runbook the following :
> 
> * Set the environment variable NAMESPACE
> 
> ```
> export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
> ```
> 
> * Then use it in other commands, for example
> 
> `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api`

Update

**Pull request merged**